managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -683,6 +683,14 @@ public void deleteCursorFailed(ManagedLedgerException exception, Object ctx) {
     }
 
     @Override
+    public ManagedCursor newNonDurableCursor(Position startCursorPosition) throws ManagedLedgerException {
+        checkManagedLedgerIsOpen();
+        checkFenced();
+
+        return new NonDurableCursorImpl(bookKeeper, config, this, null, (PositionImpl) startCursorPosition);
```

Comment:
for debugging, should broker generate or expect cursor-name or id rather passing null?